core/trino-main/src/main/java/io/trino/sql/analyzer/Scope.java

```diff
@@ -13,7 +13,6 @@
  */
 package io.trino.sql.analyzer;
 
-import com.google.common.base.Predicate;
```

Comment:
There are other like that. Can you please grep and remove all? I think we should have codestyle rule for that. Same for Supplier.

Reply:
I've already checked. There are Predicate's that are used along with other Guava APIs that accept them.